### ISSUE TITLE
Access data served via xcube serve

### ISSUE THREAD

**Issue by AliceBalfanz** [dcs4cop/xcube#717]
**Is your feature request related to a problem? Please describe.**
I would like to access datasets served using xcube server via store framework in order to list the served cubes and then test them to ensure sanity of operational services. 

**Describe the solution you'd like**
I would like to access the datasets via a new store: 

```
store = new_data_store('s3', 
                       root='s3bucket', 
                       storage_options=dict(anon=True, 
                                            client_kwargs=dict(endpoint_url="https://bc-api.brockmann-consult.de/api")))
```
and then receive identifiers of the cubes by calling: 

`list(store.get_data_ids())`

Then I also want to open the cubes, and if there is a `~` in the identifier, this should not be a problem.

**Comment by forman** [comment]
This nice idea we also already presented in the DeepESDL project in order to provide cubes processed on-the-fly, hence representing a data cube generator service.

**Comment by forman** [comment]
This is now implemented in xcube 0.13.x
